packages/components/src/styles/components/form/super-select.scss

```
@@ -0,0 +1,490 @@
+      background-position: 15px 10px;
+      background-size: var(--token-form-select-background-image-size) var(--token-form-select-background-image-size);
+    }
+
+    &[aria-current="true"],
+    &:hover {
+      color: var(--token-color-foreground-action-hover);
+    }
+
+    &:active {
+      color: var(--token-color-foreground-action-active);
+    }
+  }
+}
+
+// MULTIPLE SELECT
+.hds-form-super-select-multiple .ember-basic-dropdown {
+
+  // Options Item
+  .ember-power-select-option:not(.ember-power-select-option--no-matches-message) {
+    position: relative;
+
+    // pseudo checkbox
+    &::before {
+      position: absolute;
+      top: 9px;
+      left: 15px;
+      display: block;
+      width: var(--token-form-checkbox-size);
+      height: var(--token-form-checkbox-size);
+      background-position: center center;
+      background-size: var(--token-form-checkbox-background-image-size) var(--token-form-checkbox-background-image-size);
+      border-style: solid;
+      border-width: var(--token-form-checkbox-border-width);
+      border-radius: var(--token-form-checkbox-border-radius);
+      cursor: pointer;
+      content: "";
+    }
+
+    // unchecked checkbox:
+    &:not([aria-selected="true"]) {
+      &::before {
+        background-color: var(--token-form-control-base-surface-color-default);
+        border-color: var(--token-form-control-base-border-color-default);
+        box-shadow: var(--token-elevation-inset-box-shadow);
+      }
+    }
+
+    // unchecked checkbox, hovered:
+    &[aria-current="true"],
+    &:hover:not([aria-selected="true"]) {
+      &::before {
+        background-color: var(--token-form-control-base-surface-color-hover);
+        border-color: var(--token-form-control-base-border-color-hover);
+      }
+    }
+
+    // unchecked checkbox, disabled:
+    // NOTE: may not be useful for this component
+    &[aria-disabled="true"]:not([aria-selected="true"]) {
+      &::before {
+        background-color: var(--token-form-control-disabled-surface-color);
+        border-color: var(--token-form-control-disabled-border-color);
+        box-shadow: none;
+        cursor: not-allowed;
+      }
+    }
+
+    // checked checkbox:
+    &[aria-selected="true"] {
+      &::before {
+        background-color: var(--token-form-control-checked-surface-color-default);
+        background-image: var(--token-form-checkbox-background-image-data-url);
+        border-color: var(--token-form-control-checked-border-color-default);
+      }
+
+      // checked checkbox, hovered:
+      &[aria-current="true"],
+      &:hover[aria-selected="true"] {
+        &::before {
+          background-color: var(--token-form-control-checked-border-color-default);
+          border-color: var(--token-form-control-checked-border-color-hover);
+        }
+      }
+
+      // checked checkbox, disabled:
+      // NOTE: may not be useful for this component
+      &[aria-disabled="true"][aria-selected="true"] {
```

maybe is redundant because `[aria-selected="true"]` is already in the parent selector at line 415?
```suggestion
      &[aria-disabled="true"] {
```